### PR TITLE
[WIP - do not merge] get last_accessioned_version from preservation

### PR DESCRIPTION
**Pull request by ndushay**
DO NOT MERGE -- I'm working on the PR that will use the new preservation-client gem instead.

get `last_accessioned_version` from preservation rather than dor-services-app proxying for sdr-services-app.

Note that CodeClimate has our coverage at 97% but it also didn't have coverage stats for helpers.  I tweaked the SimpleCov config to hopefully fix this.

### CODE REVIEW

**Comment by ndushay** [comment]
Note to self - verify this after deployed to prod (objects in stage are not properly "preserved" per preservation catalog, so this is impossible to check in stage without heroics).

Reverting is a one liner in dor_object_helper.

File: app/helpers/dor_object_helper.rb [hunk 1]
```diff
@@ -56,7 +56,7 @@ def has_been_accessioned?(pid)
   end
 
   def last_accessioned_version(pid)
-    Dor::Services::Client.object(pid).sdr.current_version
+    PreservationClient.current_version(pid)
```
Comment (jcoyne) [inline app/helpers/dor_object_helper.rb]: Should PreservationClient be extracted to a Gem.  This is something dor-services-app could also use.

File: config/settings.yml [hunk 2]
```diff
@@ -51,6 +51,9 @@ DOR_SERVICES:
   # To generate the token: docker-compose run dor-services-app rake generate_token
   TOKEN: eyJhbGciOiJIUzI1NiJ9.eyJzdWIiOiJGb28ifQ.-BVfLTW9Q1_ZQEsGv4tuzGLs5rESN7LgdtEwUltnKv4
 
+PRESERVATION_CATALOG:
```
Comment (jcoyne) [inline config/settings.yml]: Please lowercase these. The config gem documents the correct usage as lowercase: https://github.com/railsconfig/config#embedded-ruby-erb We have lowercase variables already in this file. If we use uppercase the config gem ends up creating methods with the ruby convention for constants, which is confusing.

File: app/helpers/dor_object_helper.rb (hunk 1 above)
Comment (ndushay) [reply app/helpers/dor_object_helper.rb]: Per discussion after standup this morning:  we'll extract it to a gem after we have a second use case.

File: config/settings.yml (hunk 2 above)
Comment (ndushay) [reply config/settings.yml]: will do.  Actually, already done in PR #1603, and will rebase this afterwards.

**Comment by ndushay** [comment]
Closing this in favor of #1658

**Pull request closed without merging**